src/components/Search.js

```diff
@@ -0,0 +1,19 @@
+const Search = ({ setSearchTerm }) => {
+  return (
+    <>
+      <form>
+        <label htmlFor="search-input">Filter Items</label>
+        <br />
+        <input
+          type="search"
```

Comment:
reformat this file with prettier. Then this string return " to '.

Reply:
I check my IDE. My IDE has eslint configurer. Our prettier doenst have this rule.

Reply:
Thank you @elitcenk for your review!